enterprise/signals.py

```diff
@@ -68,7 +68,7 @@ def handle_user_post_save(sender, **kwargs):  # pylint: disable=unused-argument
 
     # link PendingEnterpriseCustomerUser to the EnterpriseCustomer and fulfill pending enrollments
     for pending_ecu in pending_ecus:
-        enterprise_customer_user = pending_ecu.link_pending_enterprise_user(
+        enterprise_customer_user = pending_ecu.link_pending_enterprise_users(
```

Comment:
Heads up - you changed the method name here but not in the declaration. I don't think that was intentional.

Reply:
Ah yeah, good catch. I meant to change the method name back to what it was. I had renamed it when I was thinking the method was going to need to change to return multiple, which turned out to be unnecessary.

I will rename it back to what it was :)